psl-clj/template/resources/leiningen/new/psl_clj/core.clj

```diff
@@ -0,0 +1,97 @@
+(ns {{namespace}}
+  "Example PSL Clojure program."
+  (:require
+   [incanter.core :as in]
+   [clojure.java.io :as io]
+   [psl-clj.core :as psl]
+   [clj-util.core :as u]
+   )
+  (:import
+   [org.linqs.psl.model.term ConstantType]
+   [org.linqs.psl.database.rdbms.driver H2DatabaseDriver H2DatabaseDriver$Type]
+   ))
+
+;;; Predicates
+
+(defn VOCAB
```

Comment:
Can we boost this up to one of our standard PSL examples?
Even just the simple "simple-acquaintances" one would be good:
https://github.com/linqs/psl-examples/tree/master/simple-acquaintances

Reply:
Agreed.  I'm still working on this.

Reply:
I finished upgrading the example to use simple-acquaintances.  See PR2.